vllm/sequence.py

```diff
@@ -235,6 +235,12 @@ def __init__(
     def lora_int_id(self) -> int:
         return self.lora_request.lora_int_id if self.lora_request else 0
 
+    def get_output_text(self, buffer_length: int):
```

Comment:
Just a nit/suggestion, probably rename this function since we have `seq.output_text` and `seq.get_output_text` but they actually don't return the same thing, WDYT?

Reply:
@ywang96 I think that's a good suggestion but having trouble thinking of an alternative name. Maybe `get_ready_output_text()` or `get_output_text_to_return()`? WDYT?

Reply:
`get_output_text_to_return()` sounds better to me! Again this is just a nit to make it clearer so developers won't mistakenly assume this is an class API that simply returns `self.output_text`.